### ISSUE TITLE
Microphone translators pickup emotes prefaced with the ! prefix.

### ISSUE THREAD

**Issue by scheveningensun** [Aurorastation/Aurora.3#5918]
![image](https://user-images.githubusercontent.com/11053780/50740917-47f4d280-11c4-11e9-9b59-423c55ed7f8a.png)

**Comment by Sindorman** [comment]
That's sort of normal, people could be like `! his voice changes deeper "Stop right there"`

**Comment by Menown** [comment]
Not a bug. Emotes with ! are audible emotes. Stop using non-audible emotes as audible emotes or I slap, pls.

**Comment by VTCobaltblood** [comment]
This *is* a bug - the microphone picks it up and _translates_, triggering the translation pin.

**Comment by OneOneThreeEight** [comment]
> That's sort of normal, people could be like `! his voice changes deeper "Stop right there"`

Yes, and it will translate cases such as that and print them out directly.

I don't think this is intended behavior, meaning it's a bug.
